test/collection_test.js

```diff
@@ -869,20 +879,6 @@ describe("Collection", () => {
             _status: "synced"
           });
       });
-
-      describe("Error handling", () => {
-        it("should expose per-record import errors", () => {
-          const err1 = new Error("err1");
-          const err2 = new Error("err2");
-          sandbox.stub(articles, "create")
-            .onCall(0).returns(Promise.reject(err1))
-            .onCall(1).returns(Promise.reject(err2));
-
-          return articles.pullChanges(result)
-          .then(res => res.errors)
-          .should.become([err1, err2]);
-        });
-      });
     });
```

Comment:
This test was moved to the 'should expose failing operation errors' test, right?